src/app/shared/dialog-box/dialog-box.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, OnInit, ChangeDetectionStrategy, Inject } from '@angular/core';
+import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
+
+export interface DialogData {
```

Comment:
it does not seem to be used anywhere else, should we export it?
If we are planning or reusing that for other components maybe it should be extracted from this file and imported from a common interface file?